### PR TITLE
HL-145 Part 1/2

### PR DESCRIPTION
**Pull request by pondpiu**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-145)

- add my play toggle
- show my play stats once toggled ( data query is not include in this PR, will follow in part 2 )
- open connect wallet modal if no wallet connected

image
image
image

### CODE REVIEW

**Review by bitcoinbrisbane — APPROVED** [review]

File: src/pages/Dashboard/Dashboard_Logic.tsx [hunk 1]
```diff
@@ -43,13 +60,53 @@ const Dashboard: React.FC = () => {
     return `${_time.toString()} hr`;
   };
 
+  const onMyPlayToggle = () => setMyPlayEnabled(prev => !prev);
```
Comment (brettvniekerk) [inline src/pages/Dashboard/Dashboard_Logic.tsx:63]: can also just do `setMyPlayEnabled(!myPlayEnabled)`

File: src/pages/Dashboard/Dashboard_Logic.tsx [hunk 2]
```diff
@@ -43,13 +60,53 @@ const Dashboard: React.FC = () => {
...
+      stat: stats?.performance
+        ? `${formatToTwoDecimals(stats.performance.toString())}%`
+        : undefined
+    }
+  ];
+
+  const myStatsArray = [
+    {
+      name: "Deposits",
+      stat: undefined
```
Comment (brettvniekerk) [inline src/pages/Dashboard/Dashboard_Logic.tsx:89]: ?

File: src/pages/Dashboard/Dashboard_View.tsx [hunk 3]
```diff
@@ -89,13 +72,19 @@ const DashboardView: React.FC<Props> = (props: Props) => {
                   {stat.name}
                 </dt>
                 <dd className="mt-1 text-3xl font-semibold text-gray-900">
-                  {stat.stat}
+                  {stat.stat || <Loader />}
```
Comment (brettvniekerk) [inline src/pages/Dashboard/Dashboard_View.tsx:75]: could use the new `Card` component for this

**Review by brettvniekerk — APPROVED** [review]
Merge in the latest dev branch and use the Card component for data - otherwise LGTM

File: src/pages/Dashboard/Dashboard_Logic.tsx [hunk 4]
```diff
@@ -43,13 +60,53 @@ const Dashboard: React.FC = () => {
     return `${_time.toString()} hr`;
   };
 
+  const onMyPlayToggle = () => setMyPlayEnabled(prev => !prev);
+
+  const overallStatsArray = [
+    {
+      name: "Total Liquidity",
+      stat: stats?.tvl
+        ? `$${formatNumberWithCommas(ethers.utils.formatEther(stats.tvl))}`
```
Comment (bazmatic) [inline src/pages/Dashboard/Dashboard_Logic.tsx:69]: Should be `formatUnits` since some tokens use different decimal values (eg USDT)

**Review by bazmatic — APPROVED** [review]
Nice work

**Review by bazmatic — APPROVED** [review]

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**